.dotnet/src/Custom/Moderations/OpenAIModerationsModelFactory.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenAI.Moderations;
+
+/// <summary> Model factory for models. </summary>
+public static partial class OpenAIModerationsModelFactory
+{
+    /// <summary> Initializes a new instance of <see cref="OpenAI.Moderations.ModerationCategories"/>. </summary>
+    /// <returns> A new <see cref="OpenAI.Moderations.ModerationCategories"/> instance for mocking. </returns>
+    public static ModerationCategories ModerationCategories(bool hate = default, bool hateThreatening = default, bool harassment = default, bool harassmentThreatening = default, bool selfHarm = default, bool selfHarmIntent = default, bool selfHarmInstructions = default, bool sexual = default, bool sexualMinors = default, bool violence = default, bool violenceGraphic = default)
+    {
+        return new ModerationCategories(
+            hate,
+            hateThreatening,
+            harassment,
+            harassmentThreatening,
+            selfHarm,
+            selfHarmIntent,
+            selfHarmInstructions,
+            sexual,
+            sexualMinors,
+            violence,
+            violenceGraphic);
```

Comment:
nit: For consistency, call the constructor that receives the serializedAdditionalRawData even if it's just to pass null. We never customize that constructor, so if a new property is added in the future, the generator will add it to that constructor and this call will fail. It's a way to remind us that we need to add the new property here too.

Reply:
Good call. Just updated all factories to reflect that.